### ISSUE TITLE
simple math formula confuses spell checker

### ISSUE THREAD

**Issue by ari-wwu** [streetsidesoftware/vscode-spell-checker#1955]
The second `wxyz` is not marked as "wrong" by spell checker (false negative) in the following latex snippet:

```
wxyz

$p^1_1\in[0,2)$

wxyz
```

Surprisingly, removing the backslash `\` out of the math formula (`$p^1_1in[0,2)$`) or adding a closing `]` to the end of the formula (`$p^1_1in[0,2)]$`) leads to the expected behavior that both `wxyz` are marked by spell checker as "wrong".
